sdk/go.mod

```
@@ -79,3 +79,5 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240311173647-c811ad7063a7 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/opentdf/platform/ocrypto => ../ocrypto
```

Shoudln't this be lib/ocrypto?

```suggestion
replace github.com/opentdf/platform/ocrypto => ../lib/ocrypto
```